Automattic/jetpack
# Stats: Deprecating old Stats experience

Fixes https://github.com/Automattic/red-team/issues/11

## Proposed changes:

* add a banner to old Stats experience informing users that it is deprecated

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


no

## Testing instructions:






* Go to Jetpack -> Settings and disable the new experience
* Go to Jetpack -> Stats and verify that the page has a banner informing that the Old Jetpack Stats is deprecated
